demos/apps/bookinfo/bookinfo.yaml

```diff
@@ -63,6 +69,11 @@ spec:
   selector:
     app: ratings
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ratings-v1-service-account
```

Comment:
Do we expect apps to have per-version service accounts ( v1 ) ? Seems to complex/fine grained.